lib/src/compiler-path.ts

```diff
@@ -6,14 +6,34 @@ import * as fs from 'fs';
 import * as p from 'path';
 import {isErrnoException} from './utils';
 
+const isLinuxMusl = function () {
```

Comment:
Nit: document this (in particular _why_ it works consistently)

Reply:
Added a comment